service-registry/src/test/java/org/apache/servicecomb/serviceregistry/registry/TestRemoteServiceRegistry.java

```diff
@@ -141,6 +142,13 @@ public void onPullMicroserviceVersionsInstancesEvent(@Injectable ServiceRegistry
       }
     }.getMockInstance();
 
+    new MockUp<Version>() {
+      @Mock
+      boolean checkVersion(String version){
```

Comment:
why need to mock a valid version?

Reply:
as now when version is invalid, we can not registe provider